pkg/manager/namespace_test.go

```diff
@@ -198,9 +203,10 @@ func createPods(ctx context.Context, count int) ([]v1.Pod, error) {
 	return pods, nil
 }
 
-func getNamespaces(objs []v1.Pod) (namespaces []string) {
+func getNamespaces(objs []corev1.Pod) []string {
```

Comment:
FYI @joelanford I manually switched this to a set - wdyt?

Reply:
Only thing I can think of here is that a set means that there's no potential for a test that uses this function and includes duplicate namespaces in the WATCH_NAMESPACES list.

Not sure if that was or should be a consideration?

Reply:
I assumed we'd never want the same namespace listed twice?